examples/medplum-patient-intake-demo/src/App.tsx

```diff
@@ -9,15 +17,21 @@ import { SearchPage } from './pages/SearchPage';
 import { SignInPage } from './pages/SignInPage';
 import { UploadDataPage } from './pages/UploadDataPage';
 import { IntakeFormPage } from './pages/IntakeFormPage';
+import { QuestionnaireCustomizationPage } from './pages/QuestionnaireCustomizationPage';
+import { IntakeQuestionnaireContext } from './Questionnaire.context';
 
 export function App(): JSX.Element | null {
   const medplum = useMedplum();
   const profile = useMedplumProfile();
 
+  const [questionnaires] = useSearchResources('Questionnaire', { name: 'patient-intake' });
```

Comment:
I used the name `'patient-intake'` because that's how it's set in our core data, LMK if you'd like to change it.